tools/projmgr/docs/SystemArchitecture.md

```diff
@@ -0,0 +1,64 @@
+# CMSIS Project Manager - System Architecture [Draft]
+
+## Table of Contents
+
+[Introduction](#introduction)
+
+[Classes Overview](#classes-overview)
+
+[Data Flow](#data-flow)
+
+[Layers Combinations](#layers-combinations)
+
+
+## Introduction
+
+The CMSIS Project Manager is a C++ utility provided as binary and as library with interfaces for the most common programming languages and platforms.
+It leverages open source C++ libraries available in the [Open-CMSIS-Pack devtools](https://github.com/Open-CMSIS-Pack/devtools) repository.
+The tool assists the embedded software developer in the project creation by exposing available features
```

Comment:
Unclear to me how the CLI really helps the developer.
To me, as it is written today, it is mainly a CLI to generate a cprj ?

It is only listings packs, components, dependencies...it does not resolve anything, does it ?